underfs/hdfs/src/main/java/alluxio/underfs/hdfs/acl/SupportedHdfsAclProvider.java

```diff
@@ -67,18 +70,76 @@ public AccessControlList getAcl(FileSystem hdfs, String path) throws IOException
     return acl;
   }
 
-  private AclEntryType getAclEntryType(AclEntry entry) throws IOException {
+  @Override
+  public void setAcl(FileSystem hdfs, String path, AccessControlList acl) throws IOException {
+    // convert AccessControlList into hdfsAcl
+    List<AclEntry> aclSpecs = new ArrayList<>();
+
+    for (alluxio.security.authorization.AclEntry entry : acl.getEntries()) {
+      AclEntry hdfsAclEntry = getHdfsAclEntry(entry);
+      aclSpecs.add(hdfsAclEntry);
+    }
+    // set hdfsAcl;
+    hdfs.setAcl(new Path(path), aclSpecs);
```

Comment:
if hdfs did not enable acls, does this throw an exception?

Reply:
yes, it throws an `UnsupportedOperationException`

Reply:
Should we just catch it and ignore it? If HDFS did not enable it, we should not be throwing an exception, right?

Reply:
done